### PR TITLE
Create 0000-01-02-chrisaddington96.md

### PR DESCRIPTION
**Pull request by DoggyBoy666**
Wow

### CODE REVIEW

**Review by DoggyBoy666 — COMMENTED** [review]
Woow

**Review by DoggyBoy666 — COMMENTED** [review]
Wooow

File: _posts/0000-01-02-chrisaddington96.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Wow
```
Comment (chrisaddington96) [inline _posts/0000-01-02-chrisaddington96.md]: Big Wow

**Review by DoggyBoy666 — COMMENTED** [review]
Big Woow

File: _posts/0000-01-02-chrisaddington96.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+much wow
```
Comment (chrisaddington96) [inline _posts/0000-01-02-chrisaddington96.md]: aaaaaye

**Review by DoggyBoy666 — COMMENTED** [review]
aye

**Pull request merged**